storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1895,28 +1896,28 @@ static bool innodb_init_param(void) {
   return the associated srv_page_size_shift. */
   srv_page_size_shift = innodb_page_size_validate(innobase_page_size);
   if (!srv_page_size_shift) {
-    msg("xtrabackup: Invalid page size=%llu.\n", innobase_page_size);
+    xb::error() << "xtrabackup: Invalid page size=" << innobase_page_size;
     goto error;
   }
   srv_page_size = innobase_page_size;
 
   /* Check that values don't overflow on 32-bit systems. */
   if (sizeof(ulint) == 4) {
     if (xtrabackup_use_memory > UINT_MAX32) {
-      msg("xtrabackup: use-memory can't be over 4GB"
-          " on 32-bit systems\n");
+      xb::error() << "xtrabackup: use-memory can't be over 4GB"
```

Comment:
```suggestion
      xb::error() << "use-memory can't be over 4GB"
```

Reply:
aren't we missing a `goto error;` here ?

Reply:
good catch